hystrix/src/main/java/feign/hystrix/HystrixInvocationHandler.java

```diff
@@ -106,6 +102,34 @@ protected Object getFallback() {
     return hystrixCommand.execute();
   }
 
+  private HystrixCommand.Setter config(final Object proxy, final Method method) {
```

Comment:
probably what we want to do, is make something like MethodMetadata for Hystrix. That way, parsing annotations is separate from using them. For example, we don't want to replicate javanica. We can have a simple way to specify command and group names, and leave any further settings to either out-of-band configuration or another annotation system.